src/openms/source/FORMAT/MascotGenericFile.cpp

```diff
@@ -318,10 +318,8 @@ namespace OpenMS
     }
     if (spec.size() >= 10000)
     {
-      String msg = "Spectrum to be written as MGF has " + String(spec.size()) +
-        " peaks; the upper limit is 10,000. Only centroided data is allowed - this is most likely profile data.";
-      throw Exception::IllegalArgument(__FILE__, __LINE__, OPENMS_PRETTY_FUNCTION,
-                                       msg);
+      OPENMS_LOG_WARN << "Spectrum to be written as MGF has " + String(spec.size()) +
```

Comment:
MGF file format actually mandates less than 10k peaks,
see http://www.matrixscience.com/help/data_file_help.html -->
```
There is an upper limit of 10,000 peaks per individual MS/MS spectrum.
If you see an error message reporting that this limit has been exceeded,
 it almost certainly means that your data are profile data, and not peak lists.
 It is very unlikely that a single MS/MS spectrum could ever contain more than 
1000 genuine peaks, never mind 10,000.```

Reply:
see https://github.com/OpenMS/OpenMS/issues/4655

Reply:
I can revert it, but IMHO this is just plain arbitrary.

Reply:
but its their format...

and 10k peaks in an MS2 spectrum... this has to be wrong...